lib/private/Updater/VersionCheck.php

```diff
@@ -130,4 +126,25 @@ protected function getUrlContent($url) {
 		$response = $client->get($url);
 		return $response->getBody();
 	}
+
+	private function computeCategory() {
+		$categoryBoundaries = [
+			100,
+			500,
+			1000,
+			5000,
+			10000,
+			100000,
+			1000000,
+		];
+
+		$nbUsers = $this->userManager->countSeenUsers();
+		foreach ($categoryBoundaries as $categoryId => $boundary) {
+			if ($nbUsers <= $boundary) {
+				return $categoryId;
+			}
+		}
+
+		return $categoryId + 1;
```

Comment:
I did not even know the foreach var was available to outside scope, it would be way clearer to me if it was not using it.
Maybe:
```suggestion
		return count($categoryBoundaries);
```

Reply:
I'll change it :)